src/Codeception/Snapshot.php

```diff
@@ -54,7 +69,11 @@ protected function load()
      */
     protected function save()
     {
-        file_put_contents($this->getFileName(), json_encode($this->dataSet));
+        $fileContents = file_get_contents($this->getFileName());
```

Comment:
This line looks like mistake, this method saves contents of `$this->dataSet`, not re-encodes existing file.